plugin/evm/limitorders/memory_database.go

```diff
@@ -45,6 +45,7 @@ type LimitOrder struct {
 	UserAddress             string   `json:"user_address"`
 	BaseAssetQuantity       *big.Int `json:"base_asset_quantity"`
 	FilledBaseAssetQuantity *big.Int `json:"filled_base_asset_quantity"`
+	Salt                    *big.Int `json:"salt"`
```

Comment:
this also needs to be set when `Add` functions is called at the time of OrderPlaced event

Reply:
it is being set in https://github.com/hubble-exchange/subnet-evm/blob/9ff003a6f872a868f1ef7967b3751b8577f40090/plugin/evm/limitorders/contract_events_processor.go#L96